MEME/MEME/Model/ModelReservationView/ModelReservationChartView/ModelReservationChartTableViewController.swift

```diff
@@ -0,0 +1,307 @@
+//
+//  ModelReservationChartTableViewController.swift
+//  MEME
+//
+//  Created by 정민지 on 1/20/24.
+//
+
+import UIKit
+import SnapKit
+
+class ModelReservationChartViewController: UIViewController, UIViewControllerTransitioningDelegate {
+    private var selectedButton: ModelReservationTypeButton?
+    
+    // MARK: - Properties
+    private let backButton: UIButton = {
+        let backButton = UIButton()
+        backButton.setTitle("<", for: .normal)
+        backButton.setTitleColor(UIColor.black, for: .normal)
```

Comment:
```suggestion
        backButton.setTitleColor(.black, for: .normal)
```

UIColor 생략할 수 있다는 점 참고해주시면 좋을 것 같습니다!